server

```diff
@@ -461,50 +459,25 @@ ol_install() {
   install_rpm
 }
 
-check_debian_version() {
-  case "$VERSION_ID" in
-    "9"|"10"|"11")
-      return 1
-      ;;
-    *)
-      return 0
-      ;;
-  esac
-}
-
-debian_install() {
-  if check_debian_version = 0; then
-    echo "Debian $VERSION_ID is not supported by this installer."
-    exit 1
-  fi
-  export DEBIAN_FRONTEND=noninteractive
-  ret=0
-  run_cmd curl -f -s -L -o /etc/apt/sources.list.d/scylla.list "$SCYLLA_URL" || ret=$?
-  if [ $ret -ne 0 ]; then
-    echo "Your OS is not supported by this installer because there is no package repository."
-    exit 1
+check_debian_ubuntu_version() {
```

Comment:
what about adding the centos check to this function as well?
(and name it `is_distro_supported`)

Reply:
In that case we should add all the options inside. including rhel, amzn adn oracle

Reply:
I think it will be better to add it as part of different PR and combine all the possible OSs check together in one function